apps/system/js/task_manager.js

```diff
@@ -120,7 +120,7 @@
   TaskManager.prototype.EVENT_PREFIX = 'taskmanager';
   TaskManager.prototype.name = 'TaskManager';
 
-  TaskManager.prototype.setHierarchy = function() {
+  TaskManager.prototype.setFocus = function() {
```

Comment:
Since we had removed the setHierarchy, the behavior may become wrong. Should we create another function for setHierarchy?